fixtures/7.service.sns-subscriptions-trigger.json

```diff
@@ -0,0 +1,39 @@
+{
+  "service": "canary-deployments-test",
+  "custom": {
+    "deploymentSettings": {
+      "stages": [
+        "dev"
+      ]
+    }
+  },
+  "functions": {
+    "bello": {
+      "handler": "handler.bello",
+      "events": [
+        {
```

Comment:
Could you change the `events` property so that it corresponds with a SNS subscription event rather than a CW Event? We're not using those `events` to figure out the event sources we should modify, but rather we're looking at the CloudFormation template, therefore tests pass anyway. However, I suppose it would be cool that this file reflected a real scenario.

Reply:
yea, you are right, i changed it to subscription to sns topic, instead